umqtt.py

```diff
@@ -62,8 +62,9 @@ def set_last_will(self, topic, msg, retain=False, qos=0):
         self.lw_qos = qos
         self.lw_retain = retain
 
-    def connect(self, clean_session=True):
+    def connect(self, clean_session=True, timeout=5):
```

Comment:
How do we able to use this timeout from instance of TBDeviceMqtt? Please add timeout to connect method of TBDeviceMqtt as well.